### PR TITLE
Vj/907 dictionary renderer

### PR DESCRIPTION
**Pull request by vGandhi**
https://devtopia.esri.com/runtime/java-se-sdk/issues/907

@tyle8552 Please review.

### CODE REVIEW

File: src/main/java/com/esri/samples/displayinformation/dictionary_renderer_graphics_overlay/DictionaryRendererGraphicsOverlaySample.java [hunk 1]
```diff
@@ -39,6 +39,11 @@
 import javafx.scene.layout.StackPane;
 import javafx.stage.Stage;
 
+/**
+ * Demonstrates how to apply a dictionary renderer to a graphics overlay and display mil2525d graphics. The dictionary
+ * renderer creates these graphics using a local mil2525d style file and a XML file with key-value attributes for each
+ * graphic.
+ */
```
Comment (ghost) [inline src/main/java/com/esri/samples/displayinformation/dictionary_renderer_graphics_overlay/DictionaryRendererGraphicsOverlaySample.java]: We don't put descriptions on classes right now. Please check if the readme needs to be updated for these changes.

**Comment by ghost** [comment]
@vGandhi Just a minor comment. Let's only merge the 2D sample and wait for the issues with the 3D sample to get fixed.

**Comment by vGandhi** [comment]
@tyle8552 Could you take another look? I removed the 3D sample based on https://devtopia.esri.com/runtime/java-se-sdk/issues/907#issuecomment-591142

**Comment by tschie** [comment]
@vGandhi Looks good. @john0005 heads up for dev site

**Comment by ghost** [comment]
@vGandhi Looks good. Heads up @john0005 for dev site.

**Comment by john0005** [comment]
@tyle8552 Thanks!

**Pull request merged**